### ISSUE TITLE
Encrypted Media Attachments for DMs [feature request]

### ISSUE THREAD

**Issue by alltheseas** [damus-io/damus#1217]
## user story
As a Damus user who uses DMs to share information privately with other users, I would like the capability to privately share DMs with a selected user, so that the information I intend to share only reaches the intended recipient.

### acceptance criteria
1. user can attach encrypted media in DMs

**Comment by alltheseas** [comment]
@suhailsaqan this is a bigger one, maybe NIP/protocol level. DM related. 👀

**Comment by jb55** [comment]
I think blossom can do this now? It's implemented on Amethyst I believe @vitorpamploma

**Comment by vitorpamplona** [comment]
Yeah, we just implemented what 0xChat had: https://github.com/nostr-protocol/nips/pull/1537 

It works well with Blossom but not as much with NIP-96 servers. You can use [Sattellite's CDN](https://satellite.earth/) to test things on.

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
